CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
 - Added test cases for the `ExportModal` component in `js/components/common`
 - Updated backend tests to use `tasty-discover`
 - Added documentation for running a subset of the backend tests
+- Deleted `app/Response/Image` file
```

Comment:
This is a simple enough change that you can condense this into a single Changelog entry. You don't need the third item (this type of update is implied by other changes), and try to reword the first and second to combine them into a single sentence